lib/suspenders/app_builder.rb

```diff
@@ -235,8 +235,8 @@ def configure_simple_form
     def configure_action_mailer
       action_mailer_host 'development', "localhost:#{port}"
       action_mailer_host 'test', 'www.example.com'
-      action_mailer_host 'staging', "staging.#{app_name}.com"
-      action_mailer_host 'production', "#{app_name}.com"
+      action_mailer_host 'staging', %{ENV["HOST"]}
+      action_mailer_host 'production', %{ENV["HOST"]}
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.